test/supervisor.js

```diff
@@ -145,6 +145,21 @@ describe('supervisor', function(done) {
     });
   });
 
+  describe('log decoration', function() {
+    var TAGS_WORKER = /^pid:\d+ worker:\d+ .+/;
+    var TAGS_SUPER = /^pid:\d+ worker:0 .+/;
+    var NO_TAGS = /^.+/;
+
+    describe('tag logs', function() {
+      var EXPECT_TAGS = [ TAGS_WORKER, TAGS_SUPER ];
+      var EXPECT_NO_TAGS = [ NO_TAGS, NO_TAGS ];
+
+      run('.', ['--cluster', '1', '--no-timestamp-supervisor', '--no-timestamp-workers', 'test/yes-app'], EXPECT_TAGS);
```

Comment:
@sam-github this matches the existing tests, and you've explicitly excluded `test/` from `npm run lint`